net/dnsfallback/dnsfallback.go

```diff
@@ -430,3 +431,67 @@ var (
 	metricRecursiveMismatches = clientmetric.NewCounter("dnsfallback_recursive_mismatches")
 	metricRecursiveErrors     = clientmetric.NewCounter("dnsfallback_recursive_errors")
 )
+
+// GetDERPHostnames returns the hostnames of DERP servers in the given region.
```

Comment:
As mentioned above: this package is really intended to be used as a last-chance "fallback" to make DNS requests if we can't even start up and fetch the DERP map, not really as a general-purpose DERP cache. I'd much prefer that we pass a DERP map into the captive portal function and not overload the package here.

Also, for slightly complicated reasons, we haven't updated the list of baked-in DERP servers in a while, which means that this will only hit our old DERP regions, which aren't even guaranteed to have the `/generate_204` endpoint.

Reply:
Removed any usage of `dnsfallback`.